OnlyR/Services/RecordingDestination/RecordingDestinationService.cs

```diff
@@ -48,7 +49,7 @@ public RecordingCandidate GetRecordingFileCandidate(
         {
             PathAndTrackNumber? result = null;
 
-            var path = Directory.Exists(folder) ? null : GenerateCandidateFilePath(folder, dt, 1);
+            var path = Directory.Exists(folder) ? null : GenerateCandidateFilePath(folder, dt, 1, optionsService.Options.Codec);
```

Comment:
If we create a recording in wav then change to mp3 and create a second recording, both filenames will have the same suffix "001" for example. I think it would be better if they were sequenced.

Reply:
Agreed. Made the change, but will test before pushing.